src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -47,22 +45,10 @@ export interface Props {
    * HTML label text
    */
   label?: string;
-  /**
-   * Slot for node to appear directly after field legend. Typically used to include a Toolip
-   */
-  legendAfter?: ReactNode;
   /**
    * String for the optional label. By default it is '(optional)'
    */
-  optionalLabel?: string;
-  /**
-   * Indicates that field is required for form to be successfully submitted
-   */
-  required?: boolean;
-  /**
-   * String for the required label to add additional information if needed.
-   */
-  requiredLabel?: string;
+  optionalLabel?: '(required)' | '*' | '(optional)';
```

Comment:
ambivalent on the prop naming, but I think we should get rid of the `'*'` option. in the a11y training they were saying it's extra cognitive load vs. just writing out the word 🙂

Reply:
Sounds good, will remove `'*'`